### DISCUSSION TITLE
DNS problem.

### DISCUSSION THREAD

**Discussion by a-F1** [academicpages/academicpages.github.io#2534] [Q&A]
Thanks for you wonderful work!
I've successfully deployed the page on GitHub, but my website still won't load. Why is this happening? It shows the error: DNS_PROBE_FINISHED_NXDOMAIN.

![image](https://github.com/user-attachments/assets/18ebb51b-94a2-4aa1-bf49-15f7c73af812)

**Comment by rjzupkoii** [comment]
What URL are you trying to access? DNS can take a bit of time to propagate but https://a-f1.github.io/ seems to be loading just fine.

**Comment by a-F1** [comment]
This seems to be caused by a network issue. When I use my home network or mobile data, I can access my personal homepage without any problems. However, when I connect to the school's network, a DNS error appears.

**Reply by rjzupkoii** [reply]
If you can access the page fine on other networks and get a DNS on another then this seems like a network problem as opposed to an issue with the template itself. If https://academicpages.github.io/ is not blocked by the school's network then you may want to reach out to the network administrators at your school to see if there's something about https://a-f1.github.io/ that's getting caught by a filter.

**Reply by a-F1** [reply]
Sure, I will try it. Thank you so much!
